src/components/user/UserProfile.js

```diff
@@ -23,7 +24,7 @@ export default function PersonalProfile() {
         <div className="text-center">
           <Avatar
```

Comment:
Let's replace this `Avatar` component also with the new `ProfileImage` component you've created; that way we also get the same cache-busting capabilities on this page where it will ensure that the most recently uploaded version of the image will be fetched from google's cache/storage/

Reply:
Great, then I'll merge!